core/src/ons/util.js

```diff
@@ -228,6 +228,35 @@ util.findFromPath = (path) => {
 };
 
 /**
+ * @param {HTMLElement} container - Page or page-container that implements 'topPage'
+ * @return {HTMLElement|null} - Visible page element or null if not found.
+ */
+util.getTopPage = container => container && (container.tagName.toLowerCase() === 'ons-page' ? container : container.topPage) || null;
```

Comment:
I think we can avoid creating public methods `ons-splitter#topPage` and `ons-toolbar#topPage` by replace `'ons-page' ? container : container.topPage` with `'ons-page' ? container : (container.topPage || container._topPage)`.
What do you think?